components/shared/nftCard/NftCardSkeleton.vue

```diff
@@ -8,9 +8,12 @@
     <div
       v-if="!hideMediaInfo"
       class="nft-media-info"
-      :class="{ 'nft-media-info__slim': slimMediaInfo }">
+      :class="{
+        'nft-media-info__slim': slimMediaInfo,
+        'nft-media-info__minimal': minimal,
+      }">
```

Comment:
do like this 

NftCard.vue

![CleanShot 2024-04-10 at 12 36 14@2x](https://github.com/kodadot/nft-gallery/assets/44554284/f4abdf8d-a20f-4fa6-b7fe-3939317d40af)